wandb/cli/cli.py

```diff
@@ -1421,10 +1423,55 @@ def docker(
     "--upgrade", is_flag=True, default=False, help="Upgrade to the most recent version"
 )
 @click.option(
-    "--edge", is_flag=True, default=False, help="Run the bleading edge", hidden=True
+    "--edge", is_flag=True, default=False, help="Run the bleeding edge", hidden=True
 )
 @display_error
-def local(ctx, port, env, daemon, upgrade, edge):
+def local(ctx, *args, **kwargs):
+    wandb.termwarn("`wandb local` has been replaced with `wandb server start`.")
+    ctx.invoke(start, *args, **kwargs)
+
+
+@cli.group(
+    help="Commands for operating a local W&B server", invoke_without_command=True
+)
+@click.pass_context
+@click.option("--port", "-p", default="8080", help="The host port to bind W&B local on")
+@click.option(
+    "--env", "-e", default=[], multiple=True, help="Env vars to pass to wandb/local"
+)
+@click.option(
+    "--daemon/--no-daemon", default=True, help="Run or don't run in daemon mode"
+)
+@click.option(
+    "--upgrade", is_flag=True, default=False, help="Upgrade to the most recent version"
+)
+@click.option(
+    "--edge", is_flag=True, default=False, help="Run the bleeding edge", hidden=True
+)
+def server(ctx, *args, **kwargs):
+    if ctx.invoked_subcommand is None:
+        ctx.invoke(start, *args, **kwargs)
+
+
+@server.command(context_settings=RUN_CONTEXT, help="Start a local W&B server")
+@click.pass_context
+@click.option(
+    "--port", "-p", default="8080", help="The host port to bind W&B server on"
+)
+@click.option(
+    "--env", "-e", default=[], multiple=True, help="Env vars to pass to wandb/local"
+)
+@click.option(
+    "--daemon/--no-daemon", default=True, help="Run or don't run in daemon mode"
+)
+@click.option(
+    "--upgrade", is_flag=True, default=False, help="Upgrade to the most recent version"
+)
```

Comment:
@vanpelt I think we should hide this flag and add it back if we decided not to do `wandb server upgrade`